src/chrome/chromeUtils.ts

```diff
@@ -312,3 +315,45 @@ export function getUrlRegexForBreakOnLoad(url: string): string {
     const escapedFileName = pathToRegex(fileNameWithoutExtension);
     return '.*[\\\\\\/]' + escapedFileName + '([^A-z^0-9].*)?$';
 }
+
+/**
+ * Checks if a given tcp port is currently in use (more accurately, is there a server socket accepting connections on that port)
+ * @param port The port to check
+ * @param host Optional host, defaults to 127.0.0.1
+ * @param timeout Timeout for the socket connect attempt
+ * @returns True if a server socket is listening on the given port, false otherwise
+ */
+export async function isPortInUse(port: number, host: string = '127.0.0.1', timeout = 400): Promise<boolean> {
```

Comment:
Would it be better to use localhost as a default?
Is it possible that some customer is running this on a PC with only IPv6?

Reply:
Possibly. I recall Michael mentioning that 'localhost' actually had issues on some machines, and that using '127.0.0.1' explicitly solved it. In any case, we're using the default of '127.0.0.1' in most other methods related to launching, which is why I stuck with that convention.

Reply:
Right, don't use localhost